collections/ansible_collections/massopencloud/esi/roles/network/tasks/destroy_network.yaml

```
@@ -1,7 +1,12 @@
 - name: Check that exactly one network matching the ident exists  # noqa:no-changed-when
-  ansible.builtin.command: >-
-    openstack network show {{ 'network-' + network_suffix }} -f json
-  register: network_show_cmd_raw
+  block:
+    - name: Run command to find network  # noqa:no-changed-when
+      ansible.builtin.command: >-
+        openstack network show {{ 'network-' + network_suffix }} -f json
+      register: network_show_cmd_raw
+  rescue:
+    - name: Exit because network does not exist
+      ansible.builtin.meta: end_play
```

```suggestion
      ansible.builtin.meta: end_role
```
we don't want to end the play book itself, right? https://docs.ansible.com/ansible/latest/collections/ansible/builtin/meta_module.html#parameter-free_form

---

Ah, you're right. Change made - thanks!